condition-checker/src/main/java/com/mersey/rowing/club/condition_checker/controller/openweather/OpenWeatherApiClient.java

```diff
@@ -11,11 +11,18 @@
 import org.springframework.web.client.RestClientResponseException;
 import org.springframework.web.client.RestTemplate;
 
+import java.io.*;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeFormatter;
+
 @Component
 @Slf4j
 public class OpenWeatherApiClient {
 
   RestTemplate restTemplate = new RestTemplate();
+  private static final DateTimeFormatter dtfMinusHours = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
optional, but we already import dateUtil, which has this formatter in. Could we access the formatter from there to reduce repetition, and be more maintainable? 

E.g. change (in dateutil) the following to be public 


https://github.com/AdelaideBaron/mrc-condition-checker/blob/3b431b01633173bd46a5a89b1f51b578d3de4664/condition-checker/src/main/java/com/mersey/rowing/club/condition_checker/controller/util/DateUtil.java#L18

And then access in here when needed? 

If this would make things messier though, no worries/don't bother!